lib/cloud_formation/executor.rb

```diff
@@ -107,14 +107,26 @@ def create_or_update
       when "CREATE_COMPLETE", "UPDATE_COMPLETE", "UPDATE_ROLLBACK_COMPLETE"
         update
       when "ROLLBACK_COMPLETE"
-        # ROLLBACK_COMPLETE only happens when creating stack failed
-        # The only way to solve is to delete and re-create the stack
-        raise CannotUpdateRolledbackStackException
+        try_recreate
       else
         raise UpdateInProgressException
       end
     end
 
+    def try_recreate
+      puts 'Attempting to re-create stack from ROLLBACK_COMPLETE'
```

Comment:
Can we use `Rails.logger` for emitting log statements here and the other places?

Reply:
Oh yeah good point. WIll do